pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -177,6 +186,65 @@ def destroy(self, request, repository_pk, number):
         )
         return OperationPostponedResponse([async_result], request)
 
+    def get_content_pk(self, url):
```

Comment:
get_content_unit_pk please.

Reply:
I also don't think this really belongs on the RepositoryVersion model.

Reply:
I'll update the name, but I want to leave it in here for now. We should discuss moving all these plugin API methods somewhere else.